modules/c8000v/README.md

```diff
@@ -0,0 +1,88 @@
+# Network Edge Virtual Device C8000V SubModule
+
+The Network Edge Virtual Device C8000V Module will create C8000V autonomous mode devices on the Equinix
+Network edge platform.
+
+1. Single or Non HA device
+2. HA devices
+
+Please refer to the c8000v-* examples in this module's registry for more details on how to leverage the
+submodule.
+
+<!-- Begin Module Docs (Do not edit contents) -->
+
+## Equinix Network Edge Developer Documentation
+
+To see the documentation for the APIs that the Network Edge Terraform Provider is built on
+and to learn how to procure your own Client_Id and Client_Secret follow the link below:
+[Equinix Network Edge Developer Portal](https://developer.equinix.com/catalog/network-edgev1)
+<!-- End Module Docs -->
+
+<!-- BEGIN_TF_DOCS -->
+## Requirements
+
+| Name | Version |
+|------|---------|
+| <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 1.3 |
+| <a name="requirement_equinix"></a> [equinix](#requirement\_equinix) | >= 1.34.0 |
+
+## Providers
+
+| Name | Version |
+|------|---------|
+| <a name="provider_equinix"></a> [equinix](#provider\_equinix) | >= 1.34.0 |
+
+## Modules
+
+No modules.
+
+## Resources
+
+| Name | Type |
+|------|------|
+| [equinix_network_device.cluster](https://registry.terraform.io/providers/equinix/equinix/latest/docs/resources/network_device) | resource |
+| [equinix_network_device.non_cluster](https://registry.terraform.io/providers/equinix/equinix/latest/docs/resources/network_device) | resource |
+| [equinix_network_device_platform.this](https://registry.terraform.io/providers/equinix/equinix/latest/docs/data-sources/network_device_platform) | data source |
+| [equinix_network_device_software.this](https://registry.terraform.io/providers/equinix/equinix/latest/docs/data-sources/network_device_software) | data source |
+| [equinix_network_device_type.this](https://registry.terraform.io/providers/equinix/equinix/latest/docs/data-sources/network_device_type) | data source |
+
+## Inputs
+
+| Name | Description | Type | Default | Required |
```

Comment:
License token should also be listed here if it is optional.

Reply:
listed